js/api_openlayers3.js

```diff
@@ -73,8 +73,8 @@ function init() {
   // init layers
   var lineStyle = new ol.style.Style({
     stroke: new ol.style.Stroke({
-      color: 'rgba(255, 0, 0, 0.5)',
-      width: 2
+      color: hexToRGBA(strokeColor, strokeOpacity),
```

Comment:
One more thing. The function hexToRGBA is not needed. The color may also be given in hex format. See [openlayers color](https://openlayers.org/en/latest/apidoc/ol.color.html).